src/transformers/trainer.py

```diff
@@ -4518,6 +4527,7 @@ def push_to_hub(
         blocking: bool = True,
         token: Optional[str] = None,
         revision: Optional[str] = None,
+        private: Optional[bool] = None,
```

Comment:
Would that really be helpful to have this here instead of using `hub_private_repo` in `TrainingArguments` ?